lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb

```diff
@@ -72,7 +72,7 @@ Positive pressure systems and extractors can operate as normal.
 
 
 <% if calculator.sector?("homes") %>
-  ### Other people’s homes
+  ## Other people’s homes
```

Comment:
Looking at the rest of the page, the main section title is **Cleaning** on line 1 on this file. The titles changed here ("Other people’s homes", "Restaurants, pubs, bars and takeaway services", "Shops and branches", "Vehicles") are the headers for sub-sections, and therefore should be _h3_ rather than _h2_.